src/pages/configurePatch/ConfigurePatchCore.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { useState } from "react";
+import { PageContent, PageLayout, PageSider } from "components/styles";
+import { P2 } from "components/Typography";
+import { MetadataCard } from "components/MetadataCard";
+import { StyledTabs } from "components/styles/StyledTabs";
+import { Tab } from "@leafygreen-ui/tabs";
+import { useTabs, useDefaultPath } from "hooks";
+import { CodeChanges } from "pages/patch/patchTabs/CodeChanges";
+import { paths } from "contants/routes";
+import styled from "@emotion/styled/macro";
+import { VariantsTasks, Patch } from "gql/queries/patch";
+import { css } from "@emotion/core";
+import { Input } from "antd";
+import { ConfigureTasks } from "pages/configurePatch/configurePatchCore/ConfigureTasks";
+import { ConfigureBuildVariants } from "pages/configurePatch/configurePatchCore/ConfigureBuildVariants";
+import { Body } from "@leafygreen-ui/typography";
+
+interface Props {
+  patch: Patch;
+}
+export const ConfigurePatchCore: React.FC<Props> = ({ patch }) => {
+  const { project, variantsTasks } = patch;
+  const { variants } = project;
+  const [selectedTab, selectTabHandler] = useTabs({
+    tabToIndexMap,
+    defaultTab: DEFAULT_TAB,
+    path: `${paths.patch}/${patch.id}/configure`,
+  });
+  useDefaultPath({
+    tabToIndexMap,
+    defaultPath: `${paths.patch}/${patch.id}/configure/${DEFAULT_TAB}`,
+  });
+  const [selectedBuildVariant, setSelectedBuildVariant] = useState<string>(
+    variants[0].name
+  );
+  const [selectedVariantTasks, setSelectedVariantTasks] = useState<
+    VariantTasksState
+  >(convertPatchVariantTasksToStateShape(variantsTasks));
+  const [descriptionValue, setdescriptionValue] = useState<string>(
+    patch.description
+  );
+  const onChangePatchName = (e: React.ChangeEvent<HTMLInputElement>) =>
+    setdescriptionValue(e.target.value);
+  return (
+    <>
+      <StyledBody weight="medium">Patch Name</StyledBody>
```

Comment:
What's this label for? If it's referring to the description, I think it's not needed with the ghost text that's already there

Reply:
i proposed it in spruce channel to add clarity to the input. brian, ben, myself and sam all agreed. without it the input gets lost.

Reply:
That works, but can we get rid of the ghost text then?

Reply:
done